boost-maven/boost-maven-plugin/src/it/test-dynamic-update/src/test/java/it/EndpointIT.java

```diff
@@ -75,61 +75,26 @@ public void testServlet() throws Exception {
 
             assertTrue("Unexpected response body",
                     response.contains("Hello World From Your Friends at Liberty Boost EE!"));
-        } finally {
-            method.releaseConnection();
-        }
-    }
-
-    @Test
-    public void testServletAfterCompile() throws Exception {
-        HttpClient client = new HttpClient();
```

Comment:
Hmm... just curious why we only are removing now.  I can see why we'd not include this for TomEE if it depends on "loose app"... maybe this test hadn't been running the best couple weeks?